.github/ISSUE_TEMPLATE/spec_request.md

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Request creation of a new format specification'
+labels: spec-request
+---
+<!--
+A request for a format specification consists of 2 YAML blocks followed by a free-form text description.
```

Comment:
This should explain *why* we'd want a user to start filling in some information on format in KSY form and not free-form, and not just throw "mandatory" and very formal speech like "request ... consists of ...".